README.md

```diff
@@ -1,14 +1,13 @@
 # First run
 
-Clone kpi and kobocat repos in this directory
+Clone kpi repo in this directory
```

Comment:
Maybe it could be a good idea to specify which branch it is?

Reply:
This would just be the default main (or beta) once merged right? Is there a need here?

Reply:
In my mind, it was needed until it is merged. 
Nevermind.